src/Sarif.FunctionalTests/ConverterTestData/AndroidStudio/minimalIssueWithoutPackageOrHintElement.xml.sarif

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json.schemastore.org/sarif-1.0.0",
```

Comment:
Oops. Going to have to change the logger to emit the corrected schema URL that's precisely versioned. And then regen all the test files. :P